env_sample

```diff
@@ -0,0 +1,30 @@
+# development
+DATABASE_URL=postgres:///pythondotorg
+SEARCHBOX_SSL_URL=http://127.0.0.1:9200/
+
+# development optional
+#PEP_REPO_PATH=None
+
+# production required
+SECRET_KEY=
+ALLOWED_HOSTS=127.0.0.1,
+EMAIL_HOST=
+EMAIL_HOST_USER=
+EMAIL_HOST_PASSWORD=
+EMAIL_PORT=
+DEFAULT_FROM_EMAIL=
+PEP_ARTIFACT_URL=
+FASTLY_API_KEY=
+SENTRY_DSN=
+SOURCE_VERSION=
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_STORAGE_BUCKET_NAME=
+
+
+# production optional
+#SPONSORSHIP_NOTIFICATION_FROM_EMAIL=
+#SPONSORSHIP_NOTIFICATION_TO_EMAIL=
+#AWS_DEFAULT_ACL=
+#AWS_S3_REGION_NAME=
+#AWS_S3_ENDPOINT_URL=
```

Comment:
Thanks for coming up with this file. I'm a fan of `.env` files to configure django apps as well.